firestore.rules

```diff
@@ -18,7 +18,17 @@ service cloud.firestore {
       }
 
       match /privatePlayers/{uid} {
+
+        function isCatfish(){
+          return get(/databases/$(database)/documents/lobbies/$(code)/privatePlayers/$(request.auth.uid)).data.role == "CATFISH";
+        }
+
+        function isCatfishResource(){
+          return resource.data.role == "CATFISH"
+        }
+
         allow get: if request.auth.uid == uid;
+        allow list: if isCatfish() && isCatfishResource();
```

Comment:
do we really want catfish to be able to read the entire privatePlayer document?

Reply:
They're only allowed to read other catfish private player docs in which only holds the their role(which we want them to know), stalker(wouldn't matter if they know because they're working together), and prompt in which they would have the same prompt.

Reply:
hmm I guess so but it feels wrong

Reply:
If you have a better idea lmk. We'd be only duplicating a little bit of data if you do have one